test/workers/test_websocketIOServerWorker.py

```diff
@@ -0,0 +1,40 @@
+import time
+
+import torch
+
+import syft as sy
+from syft.workers import WebsocketIOServerWorker
+
+
+def _payload(location):
+    x = torch.tensor([10, 20, 30, 40, 50.0])
+    print("About to send x")
+    x.send(location)
+    print("End of payload")
+
+
+hook = sy.TorchHook(torch)
+server_worker = WebsocketIOServerWorker(hook, "localhost", 5000, log_msgs=True, payload=_payload)
+
+
+def test_client_id():
+    android = server_worker.socketio.test_client(server_worker.app)
+    android.emit("client_id", "android")
+    assert len(server_worker.clients) == 1
+    android.disconnect()
+    server_worker.terminate()
+
+
+def test_payload_execution():
+    android = server_worker.socketio.test_client(server_worker.app)
+    android.emit("client_id", "android")
+    time.sleep(0.1)
+    android.emit("client_ack", "Android")
+    time.sleep(0.3)
+    android.emit("client_ack", "Android")
+    time.sleep(0.3)
```

Comment:
Could it be 0.1? To avoid making the tests slower.

Reply:
I tried with several values and (unfortunately) this was the minimum I could use to make the test pass
Using any `time.sleep()` in tests is usually a bad idea and we probably need to find another way to improve all those tests that are using it

Reply:
Agree. I'll create an issue for this.